lib/Backends/OpenCL/OpenCL.cpp

```diff
@@ -854,6 +860,16 @@ Error OpenCLFunction::execute(ExecutionContext *context) {
                 quantization::quantizeScaleOffset32To8(resultScale, 0);
             setKernelArg(kernel, ++numArgs, resultScaleParams);
           }
+        } else if (isa<ReluInst>(I)) {
```

Comment:
Please use `if (auto *RI = dyn_cast<ReluInst>(I)) {...}` and use ReluInst specific getters like getDest(), etc.